ga-ccd-definition/AuthorisationCaseType/AuthorisationCaseTypeGAspec.json

```diff
@@ -22,5 +22,13 @@
       "civil-administrator-basic"
     ],
     "CRUD": "R"
+  },
+  {
```

Comment:
May be we can add all "R" in one place ga-basic-access, caseworker-ras-validation and civil-administrator-basic